main.py

```diff
@@ -3,5 +1,53 @@
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
     pass
+    genres = ["Western",
+              "Action",
+              "Dramma"]
```

Comment:
There is a typo in the genre name 'Dramma'. It should be 'Drama'. This is corrected later in the code, but it's better to fix it here to avoid unnecessary updates.